worker-caf/src/main/java/com/hpe/caf/worker/AbstractWorker.java

```diff
@@ -132,7 +132,18 @@ protected final WorkerResponse createSuccessResult(final V result, final byte[]
      */
     protected final WorkerResponse createSuccessNoOutputToQueue()
     {
-        return new WorkerResponse(null, TaskStatus.RESULT_SUCCESS, new byte[]{}, getWorkerIdentifier(), getWorkerApiVersion(), null);
+        return createSuccessNoOutputToQueue("");
+    }
+
+    /**
+     * Utility method for creating a WorkerReponse that represents a success, but does not send a message to the worker's output message.
+     *
+     * @param trackTo the tracking 'trackTo' pipe to set
+     * @return a WorkerResponse that represents a success
+     */
+    protected final WorkerResponse createSuccessNoOutputToQueue(final String trackTo)
```

Comment:
This method doesn't make sense, because it hardcodes no target queue but allows a final tracking queue to be specified.  Replace it with a `createTaskCompleteResponse()` method.